securing_apps/topics.adoc

```diff
@@ -28,6 +28,7 @@ ifeval::[{project_community}==true]
 include::topics/oidc/java/servlet-filter-adapter.adoc[]
 include::topics/oidc/java/jaas.adoc[]
 endif::[]
+include::topics/oidc/java/installed-adapter.adoc[]
```

Comment:
Can you wrap this with:
    ifeval::[{project_community}==true] 
    ....
    endif::[]